locale/de/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,1250 @@
+"indicates that no information is available)"
+msgstr ""
+
+#: app/models/task.py:222
+msgid "Processing Time"
+msgstr ""
+
+#: app/models/task.py:223
+msgid ""
+"Processing node assigned to this task (or null if this task has not been "
+"associated yet)"
+msgstr ""
+
+#: app/models/task.py:223
+msgid "Processing Node"
+msgstr ""
+
+#: app/models/task.py:224
+msgid ""
+"A flag indicating whether this task should be automatically assigned a "
+"processing node"
+msgstr ""
+
+#: app/models/task.py:224
+msgid "Auto Processing Node"
+msgstr ""
+
+#: app/models/task.py:225
+msgid "Current status of the task"
+msgstr ""
+
+#: app/models/task.py:225
+msgid "Status"
+msgstr ""
+
+#: app/models/task.py:226
+msgid "The last processing error received"
+msgstr ""
+
+#: app/models/task.py:226
+msgid "Last Error"
+msgstr ""
+
+#: app/models/task.py:227
+msgid "Options that are being used to process this task"
+msgstr ""
+
+#: app/models/task.py:228
+msgid "List of available assets to download"
+msgstr ""
+
+#: app/models/task.py:228
+msgid "Available Assets"
+msgstr ""
+
+#: app/models/task.py:229
+msgid "Console output of the processing node"
+msgstr ""
+
+#: app/models/task.py:229
+msgid "Console Output"
+msgstr ""
+
+#: app/models/task.py:231
+msgid "Extent of the orthophoto"
+msgstr ""
+
+#: app/models/task.py:231
+msgid "Orthophoto Extent"
+msgstr ""
+
+#: app/models/task.py:232
+msgid "DSM Extent"
+msgstr ""
+
+#: app/models/task.py:233
+msgid "DTM Extent"
+msgstr ""
+
+#: app/models/task.py:237
+msgid ""
+"A requested action to be performed on the task. The selected action will be "
+"performed by the worker at the next iteration."
+msgstr ""
+
+#: app/models/task.py:237
+msgid "Pending Action"
+msgstr ""
+
+#: app/models/task.py:239
+msgid "A flag indicating whether this task is available to the public"
+msgstr ""
+
+#: app/models/task.py:239
+msgid "Public"
+msgstr ""
+
+#: app/models/task.py:240
+msgid ""
+"When set to a value different than -1, indicates that the images for this "
+"task have been / will be resized to the size specified here before "
+"processing."
+msgstr ""
+
+#: app/models/task.py:240
+msgid "Resize To"
+msgstr ""
+
+#: app/models/task.py:243
+msgid ""
+"Value between 0 and 1 indicating the upload progress of this task's files to "
+"the processing node"
+msgstr ""
+
+#: app/models/task.py:244
+msgid "Upload Progress"
+msgstr ""
+
+#: app/models/task.py:247
+msgid ""
+"Value between 0 and 1 indicating the resize progress of this task's images"
+msgstr ""
+
+#: app/models/task.py:248
+msgid "Resize Progress"
+msgstr ""
+
+#: app/models/task.py:251
+msgid ""
+"Value between 0 and 1 indicating the running progress (estimated) of this "
+"task"
+msgstr ""
+
+#: app/models/task.py:252
+msgid "Running Progress"
+msgstr ""
+
+#: app/models/task.py:254
+msgid "URL this task is imported from (only for imported tasks)"
+msgstr ""
+
+#: app/models/task.py:254
+msgid "Import URL"
```

Comment:
```suggestion
msgid "Import URL"
```
URL to import, or "Click to import URL"?

Reply:
Check `task.py:254` to see the context; it's a field name (mostly just displayed in the admin interface).